Context.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the DriftPHP Project
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ */
+
+declare(strict_types=1);
+
+namespace Drift\HttpKernel;
+
+/**
+ * Class Context.
+ */
+class Context
+{
+    /**
+     * Values.
+     */
+    private $values;
+
+    /**
+     * Add context value.
+     *
+     * @param string $key
+     * @param string $value
+     *
+     * @return void
+     */
+    public function set(
+        string $key,
+        string $value
+    ): void {
+        $this->values[$key] = $value;
+    }
+
+    /**
+     * Delete context key.
+     *
+     * @param string $key
+     *
+     * @return void
+     */
+    public function delete(string $key): void
+    {
+        unset($this->values[$key]);
+    }
+
+    /**
+     * Get context value.
+     *
+     * @param string $key
+     *
+     * @return string|null
+     */
+    public function get(string $key): ? string
+    {
+        return $this->values[$key] ?? null;
+    }
+
+    /**
+     * Clear context.
+     *
+     * @return void
+     */
+    public function clear(): void
+    {
+        $this->values = [];
+    }
+
+    /**
+     * Get all values.
+     *
+     * @return string[]
+     */
+    public function all(): array
```

Comment:
@mmoreram Just a small comment from me. :wink: 
The return type here will fail when you create an object like this: `$context = new Context();`. For example in [the constructor](https://github.com/driftphp/http-kernel/pull/19/files#diff-c5147ba9bd3ba0dd021e6202b2902c53R51) of the `RequestWithContext` class. Being created this way property `$values` contains `null`.

Reply:
Touché!
Thanks for that!
This class needs and deserves Unit Testing.